packages/hash/frontend/src/theme/components/inputs/MuiButtonThemeOptions.ts

```diff
@@ -14,114 +17,231 @@ export const MuiButtonThemeOptions: Components<Theme>["MuiButton"] = {
+      /** ====================== VARIANTS specific styling ============================= */
 
-        Object.assign(baseStyles, {
+      if (variant === "primary") {
+        /** ===== PRIMARY variant specific styling ===== */
+        baseStyles = {
+          ...baseStyles,
           color: theme.palette.common.white,
           background: theme.palette.blue[70],
-          backgroundColor: theme.palette.blue[70],
           position: "relative",
-          overflow: "hidden",
           zIndex: 0,
-          transition: "opacity 0.5s ease-in-out",
-          ...(size === "small" && {
-            padding: theme.spacing("8px", "20px"),
-          }),
-          ...(size === "medium" && {
-            padding: theme.spacing("12px", "28px"),
-            minHeight: 51,
-          }),
-        });
-
-        Object.assign(beforeStyles, {
+          transition: theme.transitions.create("opacity"),
+        };
+
+        beforeStyles = {
+          ...beforeStyles,
           position: "absolute",
           top: 0,
           width: "100%",
           height: "100%",
           opacity: 0,
           background:
             "linear-gradient(82.89deg, #9E56FA 5.64%, #644CFF 78.19%, #0070F4 121.05%)",
+          boxShadow: theme.boxShadows.purpleShadowMd,
+          transition: theme.transitions.create("opacity"),
           zIndex: -1,
-        });
+        };
+
+        hoverStyles = {
+          ...hoverStyles,
+          background: theme.palette.blue[70],
+        };
 
-        Object.assign(hoverBeforeStyles, {
+        hoverBeforeStyles = {
+          ...hoverBeforeStyles,
           opacity: 1,
-        });
-      } else if (variant === "tertiary_quiet") {
-        /** ===== TERTIARY button specific styling ===== */
+        };
+      } else if (variant === "secondary") {
+        baseStyles = {
+          ...baseStyles,
+          border: `1px solid ${theme.palette.blue[70]}`,
+          color: theme.palette.blue[70],
+          background: theme.palette.common.white,
+        };
+
+        hoverStyles = {
+          ...hoverStyles,
+          background: theme.palette.blue[20],
+        };
+      } else if (variant === "tertiary") {
+        /** ===== TERTIARY variant specific styling ===== */
+        baseStyles = {
+          ...baseStyles,
+          borderWidth: 1,
+          borderStyle: "solid",
+          borderColor: theme.palette.gray[30],
+          color: theme.palette.gray[80],
+          background: theme.palette.common.white,
+
+          [`& > .${buttonClasses.startIcon}, & > .${buttonClasses.endIcon}`]: {
+            color: theme.palette.gray[50],
+          },
+        };
+
+        hoverStyles = {
+          ...hoverStyles,
+          background: theme.palette.gray[20],
+          color: theme.palette.gray[90],
 
-        Object.assign(baseStyles, {
+          [`& > .${buttonClasses.startIcon}, & > .${buttonClasses.endIcon}`]: {
+            color: theme.palette.gray[80],
+          },
```

Comment:
**Blocking nit pick:** for the `:focus` state of the tertiary and tertiary_quiet variants the icon and background colour should be a lighter gray than on `:hover` according to the designs:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/42802102/158231260-4c6affc9-c375-43f9-a84c-8907f4cf9551.png">

Reply:
Ahh, I think I see the issue. When testing `:focus` state I was relying on navigating with the keyboard which rendered the right UI, but that isn't the case when `:focus` comes from clicking the button. Will update

Reply:
Addressed here => cf1381b